stable/rocketchat/values.yaml

```diff
@@ -2,8 +2,8 @@
 ## ref: https://hub.docker.com/r/rocketchat/rocket.chat/tags
 ##
 image:
-  repository: docker.io/library/rocket.chat
-  tag: 1.1.1
+  repository: rocketchat/rocket.chat
```

Comment:
Any objections to leaving the `docker.io/` part?
It's theoretically possible to configure the Docker deamon to use other registries than docker.io as default which might even pose a security issue...

Reply:
Done!